### PR TITLE
Bump meshery-operator version to the latest(v0.6.10)

### PR DESCRIPTION
**Pull request by Aisuko**
**Notes for Reviewers**

This PR fixes #

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -69,9 +69,9 @@ require (
 	gopkg.in/yaml.v2 v2.4.0
 	gorm.io/gorm v1.25.2
 	k8s.io/api v0.26.1
-	k8s.io/apiextensions-apiserver v0.26.0
```
Comment (Aisuko) [inline go.mod:72]: It comes from `go mod tidy` for checking the dependencies compatibility, a minor version bump should be ok.

File: go.mod [hunk 2]
```diff
@@ -69,9 +69,9 @@ require (
...
+	k8s.io/apiextensions-apiserver v0.26.1
 	k8s.io/apimachinery v0.26.1
-	k8s.io/client-go v0.26.0
```
Comment (Aisuko) [inline go.mod:74]: Same

**Comment by leecalcote** [comment]
@nebula-aac look good?

**Comment by nebula-aac** [comment]
@leecalcote reviewing now

**Pull request merged**